### PR TITLE
Update huawei cce installation wording

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

The original guide lacks some images in Huawei CCE, especially some UI buttons for the cluster creation process. I will go through the whole process later to add some images.

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
> Signed-off-by: Sherlock113 [sherlockxu@yunify.com](mailto:sherlockxu@yunify.com)
> 
> The original guide lacks some images in Huawei CCE, especially some UI buttons for the cluster creation process. I will go through the whole process later to add some images.

ok.

**Comment by FeynmanZhou** [comment]
@Sherlock113 May I start to review this PR?

**Comment by Sherlock113** [comment]
The test account is not usable now as it does not have any credits. Please review the PR now and ignore any UI buttons (words and format), which will be checked later once the test account is usable and I will add more images. @FeynmanZhou

File: content/en/docs/installing-on-kubernetes/hosted-kubernetes/install-ks-on-huawei-cce.md [hunk 1]
```diff
@@ -29,16 +31,19 @@ After you get the configuration file for kubectl, use kubectl command lines to v
...
 Server Version: version.Info{Major:"1", Minor:"17+", GitVersion:"v1.17.9-r0-CCE20.7.1.B003-17.36.3", GitCommit:"136c81cf3bd314fcbc5154e07cbeece860777e93", GitTreeState:"clean", BuildDate:"2020-08-08T06:01:28Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
-
 ```
 
 ## KubeSphere Deployment
 
 ### Create a custom StorageClass
 
-> Huawei CCE built-in Everest CSI provides StorageClass `csi-disk` which uses SATA (normal I/O) by default, but the actual disk that is for Kubernetes clusters is either SAS (high I/O) or SSD (extremely high I/O). So it is suggested that create an extra StorageClass and set it as default for later. Refer to the official document - [Use kubectl to create a cloud storage](https://support.huaweicloud.com/en-us/usermanual-cce/cce_01_0044.html).
+{{< notice note >}}
```
Comment (rayzhou2017) [inline content/en/docs/installing-on-kubernetes/hosted-kubernetes/install-ks-on-huawei-cce.md:40]: Is {{< notice note >}} working on ks web page? It shows as it is in GitHub page

Comment (Sherlock113) [reply content/en/docs/installing-on-kubernetes/hosted-kubernetes/install-ks-on-huawei-cce.md:40]: This format is working on KubeSphere website while not applicable in GitHub.

**Pull request merged**